examples/compiled/airport_connections.vg.json

```diff
@@ -201,7 +201,7 @@
           "opacity": {"value": 0.35},
           "stroke": {"value": "#000"},
           "description": {
-            "signal": "\"latitude: \" + (format(datum[\"latitude\"], \"\")) + \"; longitude: \" + (format(datum[\"longitude\"], \"\")) + \"; lat2: \" + (format(datum[\"lat2\"], \"\")) + \"; lon2: \" + (format(datum[\"lon2\"], \"\"))"
+            "signal": "\"\" + \"latitude\" + \": \" + (format(datum[\"latitude\"], \"\")) + \"; \" + \"longitude\" + \": \" + (format(datum[\"longitude\"], \"\")) + \"; \" + \"lat2\" + \": \" + (format(datum[\"lat2\"], \"\")) + \"; \" + \"lon2\" + \": \" + (format(datum[\"lon2\"], \"\"))"
```

Comment:
Hmm, this makes the code unnecessarily complex here. Can you think of a way to simplify the generated code?

Reply:
+1